wiki/Guides/Adding_Custom_Hitsounds/en.md

```diff
@@ -2,41 +2,52 @@
+where <sampleset> is either "normal", "soft" or "drum", and <sound> is one of the examples mentioned above, e.g. "hitclap".
 
-![Tutorial Image 1](ACH_01.png "Tutorial Image 1")
+![An example of custom hitsound naming convention inside a beatmap's Song Folder](ACH_01.png "Tutorial Image 1")
 
 ### Example 1
 
-So then you name the file (let's say the Soft clap) `soft-hitclap.wav`.
+In the example shown above, the first sound listed is a `normal-hitclap.wav`, and will replace the default hitsound that plays when a note with the Normal sampleset, hitclap addition is successfully hit. Note that this will only play on the selected sampleset—if your beatmap uses other samplesets, they will require additional hitsound files (even if you intent to use the exact same sound sample, such as is done using `soft-hitclap.wav` here.
 
-## Multiple hitsounds with the same name
+![Tutorial Image 2](ACH_02.png "Tutorial Image 2")
 
-If you want more variety and need more options for clap and all, you just add a number behind it.
+In order to have osu! play your custom hitsounds, make sure to check the second option "Custom 1" in the image shown above. Note that you do not need to add a custom hitsound for every sound in a sampleset. You will notice in the first example image that no instance of `normal-hitnormal.wav` is present, and in this case, osu! will continue to use the default sound for all successful regular hits (without hitsounds additions) when the Normal sampleset is in use.
 
-### Example 2
+## Working With Multiple Hitsounds In The Same Sampleset
 
-Like `normal-hitclap2.wav` or like `soft-hitfinish3.wav`
+Sometimes, a song might benefit from more variety in hitsounds, such as when there are many different sounds of a single instrument (e.g. a drum fill or a guitar solo) present. Alternatively, a song may have multiple sections with very different musical styles, and one group of hitsounds just doesn't fit both sections.
 
-Don't forget to go to the timing setup and change the hitsound choice to custom, and to the specific number
+In these cases, it is often helpful to utilise a different hitsound (or group of hitsounds) entirely. This can be achieved by adding a number to the end of the hitsound file name as follows:
 
-![Tutorial Image 2](ACH_02.png "Tutorial Image 2")
+`<sampleset><sound><#>.wav`
+
+where <#> can be any number between 2 and 100. Note that the first group of hitsounds does not need to be labelled with the number "1" even if multiple hitsound groups are used, so `normal-hitclap1.wav` won't work, and `normal-hitclap.wav` should continue to be used instead.
+
+In order to ensure that different-numbered hitsounds or hitsound groups play correctly, you'll need to add an inhereited timing point (green line) and change the sampleset from "Custom 1" to the option immediately below it, where you can enter the number of the hitsound group you wish to use.
+
+### Example 2
+
```

Comment:
We should get an image for this example too.

Reply:
Agreed as stated in the PR, I'll get this done tonight.

Reply:
New images added, let me know if the quality is lacking and I'll do something on a higher res monitor at work tomorrow,